README.md

```diff
@@ -1,2 +1,36 @@
 # VIPER-Templates
-Xcode files templates for various parts of VIPER module
+
+## Installation 
+```
+$ ./install.sh
+```
+
+## Templates Structure
+
+### VIPER Module Template
+
+```
+- __Module__Aseembler.swift
```

Comment:
How about `__Module__Module.swift` instead? @jasonnam used it in The 1 and I kinda like it 🤔